.github/workflows/organize_issues.yml

```diff
@@ -24,3 +24,10 @@ jobs:
         with:
           add-labels: "triage"
           repo-token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Add issue to project
+        uses: actions/add-to-project@RELEASE_VERSION
+        with:
+        #Project 10 is yb-voyager 1.0 GA
```

Comment:
Nit: Either you can drop this comment or specify the updated project name `yb-voyager`.
Nit: Space after `#`.

Reply:
Will drop the comment, not much value addition now that I think of it